doc/2/guides/introduction/2-general-purpose-backend/index.md

```diff
@@ -0,0 +1,107 @@
+---
+code: false
+type: page
+title: General Purpose Backend
+description: Kuzzle main concepts and features overview
+order: 200
+---
+
+# General Purpose Backend
+
+Kuzzle is a generic backend offering features common to all applications.
+
+Rather than redeveloping functionalities that don't bring any added value, Kuzzle proposes to use them as is to build its high-level business functionalities.
```

Comment:
I rewrited it

Rather than redeveloping functionalities that **do not bring value for your users**, Kuzzle proposes to use its basic functionalities in order to **build high-level business functionalities**.